src/transaction_builder.js

```diff
@@ -479,4 +499,18 @@ TransactionBuilder.prototype.__canModifyOutputs = function () {
   })
 }
 
+TransactionBuilder.prototype.__hasAbsurdFee = function () {
+  // not all inputs will have .value defined
+  var incoming = this.inputs.reduce(function (a, x) { return a + (x.value >>> 0) }, 0)
+
+  // but all outputs do, and if we have any input value
+  // we can immediately determine if the outputs are too small
+  var outgoing = this.tx.outs.reduce(function (a, x) { return a + x.value }, 0)
+  var fee = incoming - outgoing
+
+  // its not fool-proof, but, it might help somebody
+  // fee > 0.2BTC
+  return fee > (0.2 * 1e8)
```

Comment:
I also don't think anything beyond a hard-coded constant is reasonable without annoying real users